src/Eccube/Controller/Admin/Store/OwnerStoreController.php

```diff
@@ -259,13 +269,33 @@ public function apiInstall(Application $app, Request $request, $pluginCode, $ecc
         }
         $packageNames .= self::$vendorName.'/'.$pluginCode;
         $return = $this->composerService->execRequire($packageNames);
+
         if ($return) {
+            $url = $this->appConfig['package_repo_url'].'/collect';
+            // $composerMode = 1 is ComposerApi
+            $composerMode = 1;
+            if ($this->composerService instanceof ComposerProcessService) {
+                $composerMode = 2;
+            }
+            $data = array(
+                'code' => $pluginCode,
+                'version' => $version,
+                'download_date' => Carbon::now()->format('Y-m-d H:i:s'),
+                'core_version' => $eccubeVersion,
+                'php_version' => phpversion(),
+                'db_version' => $this->systemService->getDbversion(),
+                'os' => php_uname('s'),
```

Comment:
Maybe use: `php_uname('s') .' '. php_uname('r').' '.php_uname('v')` or `php_uname('s') .' '. php_uname('r')` for clearly because only `php_uname('s')` you can not know which version (Almost windows are windows NT!)